app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -12,7 +12,7 @@
 <%= presenter.attribute_to_html(:language, render_as: :faceted) %>
 <%= presenter.attribute_to_html(:identifier, render_as: :linked, search_field: 'identifier_tesim') %>
 <%= presenter.attribute_to_html(:keyword, render_as: :faceted) %>
-
+<%= presenter.attribute_to_html(:replaces, render_as: :faceted) %>
```

Comment:
I didnt think the handles were going to be on the show page or the form. Is that right? If so, Im not so sure this change is totally necessary.